docs/website/blog/2022-05-02-odo-v3-alpha1.md

```diff
@@ -0,0 +1,327 @@
+---
+title: odo v3-alpha1 Released
+authors:
+- name: Parthvi Vala
+  url: https://github.com/valaparthvi
+  image_url: https://github.com/valaparthvi.png
+- name: Philippe Martin
+  url: https://github.com/feloy
+  image_url: https://github.com/feloy.png
+tags: ["release"]
+slug: odo-v3-alpha1-release
+---
+
+`v3-alpha1` of odo has been released!
+
+<!--truncate-->
+
+`odo` is a tool that aims to simplify the life of developers working on cloud-native applications.
+
+Thanks to the emergence of the [Devfile](https://devfile.io/) open standard, which has been accepted as a CNCF Sandbox project (January 2022), odo v3 is now entirely based on this open standard.
+
+The goal of the Devfile standard is to define the structure of applications and how developers can work on them.
+
+A single Devfile defines the smallest building block of an application, that a developer can: build, run, test, debug and deploy. In a cloud-native environment, we generally talk about a micro-service. 
+
+Firstly, the Devfile describes the container that is needed to be deployed on a cluster during the development phases, along with the commands to execute on this container to build, run, test and debug the program, assuming the sources have been synchronized into the container.
```

Comment:
>assuming the sources have been synchronized into the container.

What does this statement mean?

Reply:
"expecting that the sources have been syncrhonized into the container (by odo)"